### ISSUE TITLE
bug and request

### ISSUE THREAD

**Issue by AMZMA** [riderkick/FMD#13]
hello, thank you for your update

mangaku seems had some bugs
1. Can't find chapter for Air Gear
2. No Tokyo Ghoul:Re in manga list

and if don't mind, can you add support for this sites too:
1. http://komikfox.net/manga-list/
2. http://mangacanblog.com/daftar-komik-manga-bahasa-indonesia.html
3. maybe later

terima kasih, terima kasih banyaaaaak :smiley_cat:

**Comment by riderkick** [comment]
It was due to their broker web page source. Our parser become confused with those broken source.
There is so many typo, something like not closed tag ">", quoted attribute ("class="name"). and many other, that make it so hard to parse. I often encounter this problem with Indonesian website, maybe it's because their "copas" habit. so their source so messed up. It was them who have to fix their source so fmd can run correctly.
Powerful browser engine can parsing those broken source so it still be able to showed in the browser.

There is already many Indonesian website. I'll not add anymore. Maybe in the future.
